modules/core/04-channel/v2/keeper/keeper.go

```diff
@@ -37,13 +41,15 @@ func NewKeeper(
 	cdc codec.BinaryCodec,
 	env appmodule.Environment,
 	clientKeeper types.ClientKeeper,
```

Comment:
Think we should still just have this interface and have `clientV2Keeper` implement it

Reply:
Do you mean have the clientV2Keeper implement the full `ClientKeeper` interface?